connect/src/model.rs

```diff
@@ -19,6 +19,12 @@ impl Deref for LoadRequest {
     }
 }
 
+#[derive(Debug)]
```

Comment:
Would it be useful to also derive `Clone` on it?

Reply:
Internally it isn't required and the previous version also didn't implement it. But I would say in terms of usefulness it might be a nice-to-have when using the library. So why not :D